src/pkgbmark.jl

```diff
@@ -126,8 +126,8 @@ function to_gist(results::PkgBenchmark.BenchmarkResults, p)
     "description": "A benchmark for Krylov repository",
```

Comment:
We ought to change this :-)

Reply:
"description": "Benchmarks uploaded by SolverBenchmark.jl" is enough generic?

Reply:
Great! Thanks.